### ISSUE TITLE
Dashboard on new tab page should be customizable

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#353]
Currently, a randomly chosen image will as a background and there are "top site" tiles being shown

Users should have the ability to:
- [x] turn off the background images https://github.com/brave/brave-browser/issues/4523
    - [x] if disabled, user should be able to customize the gradient or have a solid color (https://github.com/brave/brave-browser/issues/15252)
- [x] set your own background image https://github.com/brave/brave-browser/issues/19640
- [x] turn off the "top site" tiles https://github.com/brave/brave-browser/issues/4510
- [x] change the size or number of "top site" tiles being shown https://github.com/brave/brave-browser/issues/6619
- [x] be able to change what opening a new tab does https://github.com/brave/brave-browser/issues/2999
- [x] blank new tab page https://github.com/brave/brave-browser/issues/4509
- [x] Be able to edit and add "top sites" https://github.com/brave/brave-browser/issues/7493

(issue carried over from [browser-laptop](https://github.com/brave/browser-laptop/issues/7159))

**Comment by srirambv** [comment]
- [ ] Should also cover #404

**Comment by ExpandingMan** [comment]
It would be really nice if the new tab page also allowed you to replace the new tab "top sites" with the users own customizable list of bookmarks as in opera and vivaldi (in those browsers there is simply a special bookmarks folder which shows up in new tab as "speed dial").  Presumably this wouldn't even require changing the actual UI since the components are already there for "top sites".

**Comment by perariontaeadastra** [comment]
please introduce disabling of the tab background images. it is an issue, when working over slow Internet and Microsoft Remote Desktop Connection. Screen refresh time goes up significantly.
Thank you.

**Comment by aviaryan** [comment]
> please introduce disabling of the tab background images. it is an issue, when working over slow Internet and Microsoft Remote Desktop Connection. Screen refresh time goes up significantly.

Exactly, Brave is said to be faster than Chrome but a default like this actually acts against it. With a fast internet, this doesn't matter but when on a metered plan and in a 3G area, this means I will have to go back to Chrome.

**Comment by socketbox** [comment]
Am I naive to think that the `brave-core/components/brave_new_tab_ui/brave_new_tab.js` file, which is responsible for a good amount of a new tab's appearance, resides in a location and format that would allow it to be replaced by a non-minified and uncompressed version that the user can hack? I've searched for it locally, but can't find it.

**Comment by eljuno** [comment]
+1 from Community for "turn off" setting 
 https://community.brave.com/t/is-there-a-way-to-turn-off-homepage-new-tab-photo-wallpaper/37587?u=eljuno

**Comment by Bunderant** [comment]
I saw this mentioned in the old thread on this, but I think it's important that the "top sites" be more customizable than just display size and enabling/disabling. It's nice that they can be pinned already, but strange that the pinned items get jumbled together with the unpinned ones, especially when just getting started with Brave and there isn't enough browsing history to lend "top sites" any substantial weight. 

Bonus points if the "New Tab" tooltip actually contains the icon links for pinned sites.

**Comment by snertev** [comment]
+1, especially for a solid color background instead of image loading.

**Comment by Brave-Matt** [comment]
Another +1:
https://community.brave.com/t/how-do-i-set-a-new-tab-to-open-showing-my-home-page/39708/2

**Comment by martin-labanic** [comment]
I would really appreciate the ability to disable the top sites list.

**Comment by dhondtlaurens** [comment]
Yes, i used to have following extension on chrome (https://chrome.google.com/webstore/detail/empty-new-tab-page/dpjamkmjmigaoobjbekmfgabipmfilij) but this doesn't work on brave browser.

**Comment by tp0** [comment]
> Am I naive to think that the `brave-core/components/brave_new_tab_ui/brave_new_tab.js` file, which is responsible for a good amount of a new tab's appearance, resides in a location and format that would allow it to be replaced by a non-minified and uncompressed version that the user can hack? I've searched for it locally, but can't find it.

I can't find it locally either, so it must be loaded from some other compressed file. At least on Linux, there several multiple megabyte .pak files in `/opt/brave.com/brave/`.

**Comment by kbrafford** [comment]
I love the background image when I am on the host itself, but when I fire up brave over an RDP connection the image kills my productivity and I have to carefully remember to shrink the size of the browser before opening up a new tab.  It would be nice to set that to a solid color for accounts that I use remotely like this.

**Comment by gaelanlloyd** [comment]
Agreed, I'd like to have more control over the "new tab" window, including being able to just load `about:blank` and show nothing.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/how-to-stop-the-stupid-stats-screen-coming-up/41633/3. 

@rebron this one needs to be prioritized as well. Quite a lot of feedback from community and on this thread itself

**Comment by srirambv** [comment]
+1 from @marc1uk for blank new tab which was an option in muon

**Comment by karansapolia** [comment]
Don't know if this happens with everyone else too, but whenever I open a new tab and enter a web address  and navigate to it, first after 1-2 seconds of a light gray blank background, the Brave new tab page flashes and then the website renders. Feels like the tab window waits for the 'new tab' page to render first, blocking the requested website till then. And this usually costs extra 1-2 seconds for all new tab page navigations. Seems to be a performance problem. Please allow opening new tabs to a blank page.

**Comment by paucc** [comment]
I agree. Adding a simple option to remove the background photos and display instead a `#444444` grey colour, would be perfect.

![screen shot 2019-01-11 at 22 17 05](https://user-images.githubusercontent.com/3023717/51060128-e501a200-15ee-11e9-8d6b-e152457ba6de.png)

**Comment by karansapolia** [comment]
A simple blank page as an option without any other feature/bloat will help a lot.

**Comment by jordwalke** [comment]
The new private browsing page is really slow too.

**Comment by mailopl** [comment]
Also "about:blank" or "chrome-extension://hddnkoipeenegfoeaoibdmnaalmgkpip/toby.html#" should be allowed as a new page. This blocks me personally from transferring.

**Comment by lansalot** [comment]
Another +1 to lose the image per user setting. It's so slow over RDS, it makes opening a tab painful.

Given how much the authors trumpet the speed of Brave, it's a dumb move to introduce something that purposely slows it down. Also, if on a metered connection, there's the bandwidth aspect of course.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/start-page-thumb-shortcuts/43115 
>It’s better to add an option to set favorite site shortcuts. Hope future updates will give the ability to control shortcuts

**Comment by georles** [comment]
+1 to have option for no image at new tab

**Comment by leobel96** [comment]
+1

**Comment by edcarr** [comment]
+1

**Comment by volia2** [comment]
+1

**Comment by beccanet** [comment]
Rather than add a whole plethora of new settings, couldn't the new tab page honor the applied theme that users can already set in Settings?  I applied a theme I've been using in Chrome for a while, and it only applied the part that is in the header of the browser (the background for the tabs), not on the new tab page.  So every time I open the new tab page it's some random image (many of which are uninspiring---I care absolutely nothing about rockets and space, nor seeing roads or buildings), when I have a theme I'm content with already set, which is being ignored.

**Comment by SMargell** [comment]
Please allow for user control of the background image or the choice to NOT LOAD an image and instead default to the current theme. This is not an unreasonable ask.

**Comment by lovvtide** [comment]
+1 Please let me disable the image. Otherwise a great project!

**Comment by deniztoprak** [comment]
High quality background images should be an opt-in feature rather than default behavior. They don't add any functionality to the browser. They just consume extra bandwidth without the user's consent. It could be a nice plugin though.

**Comment by tp0** [comment]
If anyone is worried about consuring bandwidth, the images are loaded from a local location, not from the web. I verified this by extracting .pak files.

(I also manged to remove them by deleting them and re-packing the pak file. Then the the new tab page displayed a solid (purple) color background, but it is a pretty crappy hack. I don't remember the exact steps right now.)

**Comment by jangorecki** [comment]
Jus twanted to note that this is the most frequently upvoted issue on this project, having at the moment 80 upvote. Second one has only 28. I hope priorities are amended to take into account demand for that feature, that is missing comparing to old Brave version. I cannot stand those random picture at start of each new tag, that use bandwidth and memory. The more low-end device you have the less practical is to use Brave.

**Comment by conscienta** [comment]
+1000

For a project such as this, one in which speed testing is published (and great), adding images that flicker/lag upon opening a tab—an action we do countless times a day—only proves to worsen the experience. 

Baffling that it's enabled by default.

**Comment by conscienta** [comment]
Just noticed this was first requested nearly 10 months ago. Insane, it's the only thing that makes me want to ditch using it.

I wonder if those images were paid placements? Seems the only rational explanation.

**Comment by bsclifton** [comment]
Hi there folks

Just wanted to share that we ARE working on making the new tab page better (and more customizable). Here's where we're at for the moment:
- @karenkliu is working on the overall design and experience
- @cezaraugusto will be leading development on this page (he's updating shields at the moment). He'll be getting some help from a few other devs once we start

Thanks for your patience! We'll get there soon 😄

**Comment by bradleyrichter** [comment]
We understand that devs need more control and certainly need to disable images when using RDC or remote VMs. We addressed this in Muon with a simple tab setting to hide the images as well as choices for what is displayed in the new tab window. We will re-implemented these in Core first, and then continue with new tab features for customization.

(from Muon)
![image](https://user-images.githubusercontent.com/13509546/54775594-8eef4300-4bcb-11e9-9298-0bc4125ee1b0.png)

![image](https://user-images.githubusercontent.com/13509546/54775612-99a9d800-4bcb-11e9-9550-32c85007c7dc.png)


All of the images are Creative Commons licensed and we are certainly not being paid to display them. We receive countless inquires from photographers who want to contribute images to this feature and we will that in the new feature set as an opt-in approach.

Until the background image disabling lands, you can now install chrome extensions that override the new tab page. ( I confirmed this works in Version 0.61.51 release channel)

example: https://chrome.google.com/webstore/detail/blank-new-tab-page-dark/fhdjnnbfgbgheebemnjaopachfanifhh
(although it has a side effect of also blanking out the new tab ear label)

As @bsclifton said, thanks for your patience and we appreciate all of the direct feedback from the community.

**Comment by conscienta** [comment]
Great news, thanks. Please roll this one out quickly :D Other than these images this browser is exceptional.

**Comment by naryfa** [comment]
When is this option finally coming??? Is it even coming at all, or are we just being played?

**Comment by paucc** [comment]
@naryfa Hold your horses, they just said it's being developed.

**Comment by Dennisvm82** [comment]
Is there a way to adjust the time? I would like to use the 24-hour time notation.

**Comment by jordwalke** [comment]
I can't wait to switch back to Brave once this is fixed!

**Comment by Liunkae** [comment]
@Dennisvm82
> Is there a way to adjust the time? I would like to use the 24-hour time notation.
This is being tracked in #1308.

**Comment by fabianmoronzirfas** [comment]
I don't understand this. Brave is a browser that does all those nice things to protect you and your privacy and there are comments like these below:  

> I can't wait to switch back to Brave once this is fixed!

Really? This is the reason why you don't use Brave?  

> When is this option finally coming??? Is it even coming at all, or are we just being played?

What? This is an open source free of charge browser and you claim that the developers are playing you?  


>  it's the only thing that makes me want to ditch using it.

Again. What? You value your privacy lower than seeing a random image when opening an new tab?


>  I hope priorities are amended to take into account demand for that feature,

Even though this is a high frequented issue, it should not be a priority. This is a nice add-on nothing more.

> With a fast internet, this doesn't matter but when on a metered plan and in a 3G area, this means I will have to go back to Chrome.  

As some other user proved, these images are bundled with the software and don't take up any bandwidth. 

Dear Brave team. You are doing an amazing job creating a safe browser. Please don't take it personally. Please keep on doing this! Cheers 🍸

**Comment by jangorecki** [comment]
> Really? This is the reason why you don't use Brave?

If it is not usable because of amount of memory that opening a new tab requires then it is not surprising. Try some low end atom netbook having integrated gpu and 2 gb of mem.

**Comment by conscienta** [comment]
The images cause bad lag on my (very powerful) Yosemite machine, I can't click anything for 2-3 seconds when opening a new tab. Have to disable HW accel in the options to fix it.

**Comment by kbrafford** [comment]
Fabian, the issue makes the otherwise very desirable browser absolutely useless in a few cases, for a needless graphical flourish.

It’s easy to see how a developer who spends 100% of his or her time with a nice machine would never notice the issue, which is why we brought it up.

**Comment by fabianmoronzirfas** [comment]
I get the low memory problem. IMO this issue still reads like little issue description and a lot like feature requests and complaining.

**Comment by kbrafford** [comment]
It’s not just low memory.  I run Brave on a cloud host with tons of cores and 256GB of memory, and access it from a 24-core computer with 128GB, but because I have to use RDP Brave is totally useless...only because of the start screen.

**Comment by paucc** [comment]
@kbrafford You can use the workaround pointed by Brad:

> Until the background image disabling lands, you can now install chrome extensions that override the new tab page. ( I confirmed this works in Version 0.61.51 release channel)
> 
> example: https://chrome.google.com/webstore/detail/blank-new-tab-page-dark/fhdjnnbfgbgheebemnjaopachfanifhh
> (although it has a side effect of also blanking out the new tab ear label)

And please, can we limit this issue's use to only help the developers and not to ignite another online discussion?

**Comment by jangorecki** [comment]
Maybe you could just split this issue into 2+ smaller ones so blank new page could be easily rolled out without waiting months for full blown customisation of new page? Months are passing and such a simple request to fix this regression (comparing to old brave) is blocked by features that many users might not even need.

**Comment by conscienta** [comment]
> Maybe you could just split this issue into 2+ smaller ones so blank new page could be easily rolled out without waiting months for full blown customisation of dashboard? Months are passing and such a simple request to fix this regression (comparing to old brave) is blocked by features that many users might not even need.

This (x1000).

**Comment by karenkliu** [comment]
@rebron, our product manager for the New Tab page is working on splitting these into smaller issues.

**Comment by matigous** [comment]
Waiting...

**Comment by cezaraugusto** [comment]
un-assigning myself and assigning @imptrx since it's now part of the new amazing work happening in NTP. cc @rebron please coordinate as needed

**Comment by ta32** [comment]
chrome allows you to pick a single background image. this is also preferable - i don't like the idea of random images being picked for my browser home page. What if one of them is NSFW, that would be a disaster.

**Comment by tp0** [comment]
> i don't like the idea of random images being picked for my browser home page. What if one of them is NSFW, that would be a disaster.

There are 5 or so images, all landspace or similar, and contrary to an apparent popular belief, they are not fetched from the web, they are in local directory (inside a pak file).

**Comment by ta32** [comment]
> > i don't like the idea of random images being picked for my browser home page. What if one of them is NSFW, that would be a disaster.
> 
> There are 5 or so images, all landspace or similar, and contrary to an apparent popular belief, they are not fetched from the web, they are in local directory (inside a pak file).

so your saying the images picked for backgrounds aren't updated automatically? 
I understand they might be cached but if the browser randomly picks new pictures every month based on a few Instagram accounts then what i said might be possible.

**Comment by th3joker21** [comment]
I have a theme on my browser so would be nice if the image was transparent so i could see the theme behind the image. I love the stats and the time and the top recent sites.

**Comment by ShadronSoul** [comment]
In my opinion these options should have been implemented from the very beginning. They're really basic feautures and the lack of them makes my Brave experience way worse than it could be.

**Comment by bcrownover** [comment]
Making someone look at something they didn't choose is one of the biggest reasons people switch from other browsers. I'm happy with a blank tab extension, but I'm looking forward to Brave doing what Brave does best: Giving people a choice.

**Comment by idefixx** [comment]
Are you people kidding me!?!

You want to be the privacy browser and you freaking download a random pick from some website than can and will log my IP every time I open a new tab.

WTF is wrong with you!?!?

Uninstalled NEVER going to reinstall. FU

**Comment by cndouglas** [comment]
As of the latest nightly build, the background image is off by default, and there is a setting to turn it on.

![](https://user-images.githubusercontent.com/19424103/59967375-89690800-94ee-11e9-9e95-fbf6cf7e90c4.png)

**Comment by bjg222** [comment]
The image on the NTP is honestly not a big deal, especially given that it's just bundled with the browser.  I do find that 6 recent/frequent sites is not near enough, and leaves a lot of empty space on the NTP.  Rather than focus on the image being off by default (it looks far nicer with it on, to me, and an option to turn it off would help those who don't like it), I think some of the usability suggestions, like more recent/frequent sites or pinning of sites would be a more useful priority.

**Comment by bcrownover** [comment]
Strongly disagree with bjg222 on what is or is not a big deal. I find links to frequently accessed pages or pictures quite distracting when I'm trying to maintain a high degree of focus. I need my opening browser window and new page tabs to be blank. IE, Edge, and Firefox have blank tabs on my work machine. Chrome has been removed now that an extension is providing a blank page function for Brave. Had that not been possible, I probably would have removed Brave.

**Comment by nebular** [comment]
The new tab page turns my pristine Mojave OS into a windows tablet and I really feel sick just looking at it. Please, can you remove clock, icons and specially wallpapers?

**Comment by nebular** [comment]
Also would be interesting to give it an overall different look. This resembles Windows 10 with that big clock and the icon design, or even worse, that "bing" thing. Why not just a grid, something really simple?

**Comment by bsclifton** [comment]
We're making some good progress on here - shout out to @imptrx doing the work

As shown above, you can now toggle the background on/off and coming soon you'll be able to toggle each individual element also (so you can hide stats, clock, tiles, etc)

Stay tuned! I'll update the original post to link to the updated issues and check the boxes for items that have been completed

**Comment by fhelwanger** [comment]
That's very cool 🎉 

In addition to the option to show the background image or not, will we be able to choose a custom image there? Or that should be done by a extension? 🤔 

I like images, I just don't like the current ones 😄

**Comment by bsclifton** [comment]
@fhelwanger you should already be able to override the behavior of the new tab using extensions 😄

There is https://github.com/brave/brave-browser/issues/3556 which tracks being able to customize the backgrounds, but I'm not sure if we'll be able to get to this soon. However, we SHOULD be able to start work on https://github.com/brave/brave-browser/issues/2999 soon, which allows you to choose what the new tab has:
> <img width="677" alt="screen shot 2019-01-16 at 9 56 30 pm" src="https://user-images.githubusercontent.com/5951041/51298510-a9dde500-19d9-11e9-86b3-7ebb653964b1.png">
>
> a) Dashboard with images
> b) Homepage
> c) Default search engine
> d) Blank page

**Comment by bsclifton** [comment]
Another one down - huge thanks to @imptrx for https://github.com/brave/brave-core/pull/2762 😄 

(Should be available in Nightly soon!)

You can now enable/disable any element on the page.  More coming soon! Thanks for your patience, folks 😄

**Comment by ArtexJay** [comment]
Allow users to use their own image as new tab background

**Comment by ArtexJay** [comment]
> It would be really nice if the new tab page also allowed you to replace the new tab "top sites" with the users own customizable list of bookmarks as in opera and vivaldi (in those browsers there is simply a special bookmarks folder which shows up in new tab as "speed dial"). Presumably this wouldn't even require changing the actual UI since the components are already there for "top sites".

Would love to see this in Brave

**Comment by karenkliu** [comment]
Hi everyone, I'm the designer for the New Tab Page and this is roughly the order of *front-end* work that's planned with @rebron, the product manager:

1. Fix Top Tiles
- Stop pulling from user History
- Default Top Tiles the user starts with is brave.com, basicattentiontoken.org, and a third one TBD
- Remove the pin and bookmark tile options, leaving only "X" to get rid of a top site
- Allow user to add their own website URL as their top tile
- Update top tile grid so that there is no limit on how many top tiles a user can add

2. Wallpaper
- Allow users to upload their image and set as wallpaper
- Add more 'no-image' options, like gradients and setting solid colors
- Add integration with a stock image library so the user can select from there

3. Stats
- Make text color white
- Update labels to match labels in the Shields panel

Currently being fixed: Make clock widget match the user's system time format (12h, 24h).

We will be creating separate issues for these items, which addresses the OP and adds functionality beyond it. This list does not indicate the complete priority of work as there may be non-front-end work that takes precedence. As @bsclifton already mentioned, you can check the project board for updates:
https://github.com/brave/brave-browser/projects/19

**Comment by bjg222** [comment]
> * Stop pulling from user History

I realize that others here disagree with my preferences, and I am not trying to contradict those opinions, but I propose this functionality at least be maintained as an option rather than being removed completely.  I, personally, like this capability (again, before anybody gets mad at me, this is my *personal* preference, I am *not* attempting to speak for all users), and use the "most visited sites" tiles pretty often to jump back to places I go a lot.

**Comment by rebron** [comment]
@bjg222 Thanks for the feedback and datapoint for top tiles pulling from history. We'll revisit when we can add some additional functionality around it (top tiles pulling from history) as well as give users a better sense of what will display next.

**Comment by Valinwolf** [comment]
Any chance that we will have the option to set a URL as the new tab page anytime soon? It's the one thing that drives me insane about most browsers available today. About the only one that has it is Vivaldi, and maybe Firefox.

**Comment by cvoges12** [comment]
Can we add the option for a 24 hour based clock instead of 12 hour?

**Comment by cndouglas** [comment]
24 hour clock issue: #1308

**Comment by cvoges12** [comment]
@cndouglas Wow, that feature request has been around for a while now. I think the approach they have is to have it detect automatically though. You could simply have a change as an option.

**Comment by gaelbouvyer** [comment]
When opening a new tab, is it possible to have a specific url showing instead of the dashboard?

**Comment by bsclifton** [comment]
@gaelbouvyer we have an issue tracking that, if you'd like to subscribe 😄 
https://github.com/brave/brave-browser/issues/2999

It hasn't been completed yet, but it is something we want to do

**Comment by LukasBombach** [comment]
I'd like to add my own links (bookmarks) to the dashboard. From what I can read it's not possible. I'm a developer too, how can I help?

**Comment by ArtexJay** [comment]
> I'd like to add my own links (bookmarks) to the dashboard. From what I can read it's not possible. I'm a developer too, how can I help?

This would be the closest thing to Opera's/Vivaldi's implementation of speed dials which i think is the best way.

**Comment by Lissy93** [comment]
I really like the new tab page, but I just want to be able to change the photographs that load. A few of them are cool, but there are some I really don't like. I've spent ages looking through code and trying to trace where they are loaded from, but with no luck. 

Could this be a feature in the future? Alternatively, if anyone knows of the top of their head how I can modify them manually, that would be amazing

Thanks very much, and keep up the awesome work Brave Devs! 🚀

**Comment by fhelwanger** [comment]
@Lissy93 they plan to add it, check this: https://github.com/brave/brave-browser/issues/353#issuecomment-513906098

**Comment by rbABRAXAS** [comment]
hello Brave Devs, hello everyone

still nothing?

**Comment by cgcgcgcgcg** [comment]
In Vivaldi I just use the Black New Tab Page extension since I always use dark themes & styles.
This doesn't work in Brave since this Vivaldi setting simply doesn't exist in Brave.
![ksnip_20200131-075743](https://user-images.githubusercontent.com/38960370/73515899-8082d200-43ff-11ea-8200-fbc874421b8c.jpg)
This is a real deal breaker for using Brave.

**Comment by Lissy93** [comment]
@cgcgcgcgcg - I don't think this is an issue with Brave

**Comment by cgcgcgcgcg** [comment]
> @cgcgcgcgcg - I don't think this is an issue with Brave

It is on my Manjaro Linux and that's why I stopped using Brave.
It's at least problematic to not have new tab settings on a browser to change new tab behavior, and force peeps see that bloated thing every time they click on new tab.

**Comment by Lissy93** [comment]
I'm able to change it just fine, I'm on Arch which is what Manjaro is based off. I think your using an old version of Brave? It's not an isue with the current version, tested on Windows and Linux (Arch)

**Comment by k-kruglov** [comment]
I agree that themes should change the new tab page color too (or have this as an option).
When i open new tabs one after another very fast, i see that tab page has the color of the theme but then brave changes it to the default one (i have wallpaper disabled). Give us the option, please. Browser already does this, you just decided that default color of the page should be the orange one for some reason.

**Comment by onmyouji** [comment]
Kinda related to this, please allow us to enable Sponsored Images without any Background Image. Currently the 'Show Sponsored Images' settings is tied to the Background Image.

**Comment by madhums** [comment]
When can we have a simple dark background (without an image) for new tab page?

**Comment by JustWhelmed** [comment]
I prefer black backgrounds cause my eyes get strained at night. It's pretty absurd I can't do something that simple with brave.

**Comment by Valinwolf** [comment]
@JustWhelmed yeah. What gets me is you have two choices: random (and usually overly bright) images or a blinding gradient that is absurdly hideous. I want a simple dark new tab or my favorite website to show when I open a new tab, not a blinding image or gradient. However I don't want to bog down the browser or "add an extra link to the chain of trust" by adding new tab extensions.

**Comment by rafaelcastrocouto** [comment]
It was a great addition to Brave being able to customize the 'New tab' but it's past time go take a step forward. Currently we can hide each panel (rewards, stats, clock and top-sites) and choose between images (sponsored or not) and a predefined gradient.

Things that could be improved:

1. ~Change the order of the links in the top-sites~
1. Change the number of links in the top-sites
1. Resize each panel (small, medium and big would be enough)
1. Change the position of each panel
1. Clock configurations (12PM or 24h, include date, seconds, etc)
1. Choose a local folder/online gallery for the images
1. Change the color of the background gradient
1. Change the color of the text (necessary to keep contrast)
1. Add an option to "edit the new tab html/css"

I can help on this but I don't want to overlap with @bsclifton awesome work.
How can I be of service to improve this?

**Comment by vikdb** [comment]
I just want to set my own url for the new tab..

**Comment by rafaelcastrocouto** [comment]
@vikdb I think you should open a new issue for the custom URL since it's a feature that would not live inside the new tab. Instead it would have to be placed somewhere in brave://settings/ probably next to the "on startup" options.

**Comment by Valinwolf** [comment]
@rafaelcastrocouto it's already part of this issue. Check OP's list, third up linked issue #2999.

**Comment by Valinwolf** [comment]
@rafaelcastrocouto I would be happy with a "widget" based new tab, where I could select what I want to show and where to show it.

**Comment by rafaelcastrocouto** [comment]
For those willing to use a custom URL and are open to install an extension (I also think that this shouldn't need one) I recommend [Custom New Tab URL Extension](https://chrome.google.com/webstore/detail/custom-new-tab-url/mmjbdbjnoablegbkcklggeknkfcjkjia/related?hl=en) it simply does the job that should have been already done by the chromium project a long time ago.

**Comment by Valinwolf** [comment]
@rafaelcastrocouto as mentioned before, adding an extension adds an additional party you have to trust and it's a basic feature that all browsers _should_ have.

**Comment by dancojocaru2000** [comment]
Why are there suddenly e-toro ads on the new tab page?

**Comment by Valinwolf** [comment]
@dancojocaru2000 it's part of brave rewards.

**Comment by dancojocaru2000** [comment]
> @dancojocaru2000 it's part of brave rewards.

Only one problem: I don't have Brave Rewards turned on.

**Comment by Valinwolf** [comment]
@dancojocaru2000 either way, you can turn them off by clicking the slider-looking icon in the lower right corner and unchecking sponsored images.

**Comment by ashwinparthasarathi** [comment]
When it comes to customizing the new tab dashboard, I haven't found any documentation about how we can create new Widgets/Panels.

Is that something we can already do? Would love to create my own widgets with things required for me.

**Comment by musicbyjing** [comment]
Hey devs what's up

**Comment by bsclifton** [comment]
Quick update - I've submitted a PR just now to fix how top sites work:
https://github.com/brave/brave-core/pull/6584

This should unblock some of these tasks and make them MUCH easier to implement. We have a spec created by @karenkliu and @rebron that we'll look at incrementally implementing. Thanks for your patience folks 😄

**Comment by srirambv** [comment]
+1 from @specter78 for solid colour background instead of a gradient. Updated the issue in the original issue description

**Comment by aaronsantamaria** [comment]
+1 for #5838
and
+1 for #6619 
https://www.reddit.com/r/brave_browser/comments/jixyhq/dashboardnew_page_customization/

**Comment by rafaelcastrocouto** [comment]
It's developing but backwards ... 😪
Fixing top sites nor dragging them work anymore.
White icons became invisible.

No requested features delivered after 2 years:
- Can't customize background
- Can't customize top sites
- Not even a dark theme

As a bonus we get an over 500KB brave_new_tab.bundle.js 
and crypt widgets nobody asked for.

You can do better than this brave team.

**Comment by bsclifton** [comment]
@rafaelcastrocouto and others - update here about latest changes that were introduced with 1.17
https://github.com/brave/brave-browser/issues/7493#issuecomment-732430639

Now that we're integrated with instant service and other code from Chromium, we can look at pulling in the background image setting code without having to do everything from scratch

**Comment by rafaelcastrocouto** [comment]
Despite your great work @bsclifton, we DO keep getting things that were NOT requested (such as the new "Scroll for Brave Today") and NOT getting simple things that DO were requested such as:

Customize background
Customize top sites
Light/Dark themes

I really hope this will all be resolved soon with latest changes introduced by 1.17

I'll take this opportunity to add another item to the list:

Disable "Scroll for Brave Today"

**Comment by douglasparker** [comment]
I got tired of all the crap being pushed in our faces taking priority over basic browser functionality.

Brave is still an ad company after all. I switched browsers and you should too if they can't get it together and focus on what really matters: the end user experience.

**Comment by warwickmm** [comment]
> I'll take this opportunity to add another item to the list:
>
> Disable "Scroll for Brave Today"

@rafaelcastrocouto, is this what you're looking for?

![image](https://user-images.githubusercontent.com/17345532/103249590-d9ffc380-4924-11eb-9b70-9261815ed4a0.png)

**Comment by pythonmaxi** [comment]
Is there still no way to use your own background on the dashboard?

**Comment by bsclifton** [comment]
For folks watching this - we just landed the new top sites experience. You can add/edit and it looks loads better. Live on Nightly now - for some example screenshots, etc see https://github.com/brave/brave-core/pull/7859

We'll continue to make progress on these - thanks for your patience 😄

**Comment by de-cipher0** [comment]
Hey I am not sure if this is the right place to say this since I just started using git and brave but I'd like to suggest a feature to add live wallpapers as you can do in the opera browser. It's a really cool feature and actually opera is really upping their game with all these add-ons and stuff

**Comment by anujkmr953** [comment]
october 13th,still no custom background.just getting useless video call and other non essential things.Developers should focus more on Browser functionality.Android brave still sucks.Many glitches.What are developers doing!!

**Comment by Marshi7** [comment]
It's 2022... Can you just add custom backgrounds already?

**Comment by rebron** [comment]
@Marshi7 Wanted to make sure you saw https://github.com/brave/brave-browser/issues/19640. Custom background images was added to 1.37 earlier this year. 

This issue is a meta issue with a to do list of several issues. We have two more items left that we're close to finishing up.

**Comment by fallaciousreasoning** [comment]
https://github.com/brave/brave-browser/issues/6619 should land in [Nightly](https://brave.com/download-nightly/) in the next couple of days.

**Comment by rebron** [comment]
Closing. All sub-tasks have been completed.

**Comment by sammit63** [comment]
i want a little more customization as user can be able to to create a checklist if possible ?(like to do checklist )
